sdk/endpoints/online/managed/debug-online-endpoints-locally-in-visual-studio-code.ipynb

```diff
@@ -0,0 +1,197 @@
+    "    CodeConfiguration,\n",
+    "    Environment,\n",
+    ")\n",
+    "from azure.identity import DefaultAzureCredential, AzureCliCredential"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "name": "launch-container-2"
+   },
+   "outputs": [],
+   "source": [
+    "subscription_id = \"<YOUR-SUBSCRIPTION-ID>\"\n",
+    "resource_group = \"<YOUR-RESOURCE-GROUP>\"\n",
+    "workspace_name = \"<YOUR-WORKSPACE>\"\n",
+    "\n",
+    "endpoint_name = \"<YOUR-ENDPOINT-NAME>\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "name": "launch-container-3"
+   },
+   "outputs": [],
+   "source": [
+    "credential = AzureCliCredential()\n",
+    "ml_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group,\n",
+    "    workspace_name=workspace_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "launch-container-4"
+   },
+   "outputs": [],
+   "source": [
+    "deployment = ManagedOnlineDeployment(\n",
+    "    name=\"blue\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    model=Model(path=\"../../model-1/model\"),\n",
+    "    code_configuration=CodeConfiguration(\n",
+    "        code=\"../../model-1/onlinescoring\", scoring_script=\"score.py\"\n",
+    "    ),\n",
+    "    environment=Environment(\n",
+    "        conda_file=\"../../model-1/environment/conda.yml\",\n",
+    "        image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    ),\n",
+    "    instance_type=\"Standard_DS2_v2\",\n",
+    "    instance_count=1,\n",
+    ")\n",
+    "\n",
+    "# Note the local and vscode_debug flags\n",
+    "deployment = ml_client.online_deployments.begin_create_or_update(\n",
+    "    deployment, local=True, vscode_debug=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "name": "launch-container-5"
+   },
+   "source": [
+    "## 2. Debug your endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request_file_path = \"<REQUEST_FILE_PATH>\"\n",
+    "\n",
+    "endpoint.invoke(endpoint_name, request_file_path, local=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "launch-container-6"
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = ml_client.online_endpoints.get(endpoint_name, local=True)\n",
+    "endpoint.as_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Edit your endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "edit-endpoint-1"
+   },
+   "outputs": [],
+   "source": [
+    "new_deployment = ManagedOnlineDeployment(\n",
+    "    name=\"green\",\n",
```

Comment:
Personally, I think this example notebook is for debugging endpoint/deployment, so it may be better to show how to update the existing deployment instead of creating a new deployment. 
The code is just for an example so any changes will be great, like changing `instance_type` or `instance_count` and so on.